pkg/cmd/kafka/create/create.go

```diff
@@ -237,6 +239,7 @@ func runCreate(opts *options) error {
 			Name:          opts.name,
 			Region:        &opts.region,
 			CloudProvider: &opts.provider,
+			ClusterId:     *kafkamgmtclient.NewNullableString(&opts.clusterId),
```

Comment:
@dimakis this is awesome work, I'd like to us to please let's take a note that we'll need to extend this over to https://github.com/redhat-developer/app-services-cli/blob/36badbc88c93c4eef81aab174a5a76a13257ad3d/pkg/cmd/kafka/create/create.go#L449 as well.

i.e what is done here is fine for non interactive mode but we need to build on top of this by thinking on how we can going to extend this to interactive mode. We'll need to get UX involved in this.

i.e what kind of flow we want customers to have. `list existings clusters -> pick one -> etc`. That can be done in a follow PR and we'll need to involve UX on this. Let's sync with them.